ui/app/components/allocation-service-sidebar.hbs

```diff
@@ -94,6 +94,9 @@
               {{#each @service.tags as |tag|}}
                 <span class="tag is-service">{{tag}}</span>
               {{/each}}
+							{{#each @service.canary_tags as |tag|}}
+								<span class="tag canary is-service">{{tag}}</span>
+							{{/each}}
```

Comment:
Rather than make these their own conditional table column, decided to append them to the end of the rest of the `tags` on a service, should they exist.